index.html

```diff
@@ -1,12 +1,197 @@
+    <div class="mobile-navigaion-block">
+      <button type="button" aria-label="open menu" class="open-menu open">
+        <i class="fa fa-bars"></i>
+      </button>
+      <nav class="mobile-nav">
+        <button type="button" aria-label="close menu" class="close-menu close">
+          <i class="fa fa-close"></i>
+        </button>
+        <ul>
+          <li><a class="close active" href="./index.html">Home</a></li>
+          <li><a class="close" href="./about.html">About</a></li>
+          <li><a class="close" href="#">Service</a></li>
+          <li><a class="close" href="#">Blogs</a></li>
+          <li><a class="close" href="#">Support</a></li>
+          <li><a class="close" href="#">News</a></li>
+          <li class="bordered-link"><a class="close" href="#">CONTACT US</a></li>
+        </ul>
+      </nav>
+    </div>
+    <div class="desktop-navigation">
+      <div class="top-nav-bar">
+        <ul>
+          <li><a href="www.facebook/DatStarTechnology" class="social-icons"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+          <li><a href="#" class="social-icons"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+          <li><a href="t.me/datstartech3" class="social-icons"><i class="fa fa-telegram" aria-hidden="true"></i></a></li>
+          <li><a href="DatStarTechnology@gmail.com" class="social-icons"><i class="fa fa-envelope-o" aria-hidden="true"></i></a></li>
+          <li><a href="index.html" class="social-icons">My Page</a></li>
+        </ul>
+      </div>
+      <div class="header-container">
+        <div class="logo">
+          <a href="./index.html">
+            <img src="./images/logo.png" alt="DATStar Technology">
+          </a>
+        </div>
+        <nav class="desktop-nav">
+          <ul>
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#service">Service</a></li>
+            <li><a href="#blog">Blogs</a></li>
+            <li><a href="#support">Support</a></li>
+            <li><a href="#news">News</a></li>
+            <li class="bordered-link"><a href="#">CONTACT US</a></li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+  </header>
+  <main class="main-section-container">
+    <div class="transparent">
+      <div class="main-section">
+        <h2 class="title-intro">
+          <q>Welcome to DATStar Technology</q>
+        </h2>
+        <h1 class="title-info">Your Number One GPS Tracking and Security System</h1>
+        <div class="description">
+          <p>We offer a system that allows our clients to remotely control their assets. Our system assists the client in managing vehicle usage, 
+        fuel, tire performance, driver performance, and vehicle maintenance.</p>
+        </div>
+        <div class="office">
+          <h2>We are open from Monday ~ Saturday(12AM)</h2>
+          <p>You can reach us: <br> @Cabey Building 3rd floor, Room Number: 304</p>
+        </div>
+      </div>
+    </div>
+  </main>
```

Comment:
### [OTIONAL] 

- [ ] Kindly I would recommend you work on the `font -size` of this text on small screen devices. check the image screenshot below 👇  👇

![image](https://user-images.githubusercontent.com/65068771/147223488-cf353d60-7225-4843-ae8d-2882bfb24a03.png)